lib/ohai/plugins/linux/virtualization.rb

```diff
@@ -186,7 +186,8 @@ def docker_exists?
     # Full notes, https://tickets.opscode.com/browse/OHAI-551
     # Kernel docs, https://www.kernel.org/doc/Documentation/cgroups
     if File.exists?("/proc/self/cgroup")
-      if File.read("/proc/self/cgroup") =~ %r{^\d+:[^:]+:/(lxc|docker)/.+$}
+      if File.read("/proc/self/cgroup") =~ %r{^\d+:[^:]+:/(lxc|docker)/.+$} ||
+         File.read("/proc/self/cgroup") =~ %r{^\d+:[^:]+:/[^/]+/(lxc|docker)-.+$}
```

Comment:
Avoid reading twice by assigning `File.read("/proc/self/cgroup")` to a variable.